static/js/domain-registration.js

```diff
@@ -200,6 +200,9 @@
                     onboardingDomainRegistration.classList.add("is-hidden");
                     onboardingDomainRegistration.nextElementSibling.classList.add("is-visible");
 
+                    const dashboardDomainRegistrationPrompt = document.querySelector(".mpp-dashbaord-header-action");
+                    dashboardDomainRegistrationPrompt?.classList.add("is-hidden");
```

Comment:
The `?` is present here because if a user is premium/going through onboarding and HASN'T created an alias yet, there is no premium header section at all.